scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1539,6 +1539,22 @@ public void sendRecord(Record record) {
 			public void sendFlight(DTLSFlight flight) {
 				sendHandshakeFlight(flight, connection);
 			}
+
+			@Override
+			public void pauseRetransmission() {
```

Comment:
Hm, so it's about the term "retransmission". 

I understood this as "single retransmission" triggered by the `RetransmitTask `scheduled in `scheduleRetransmission`. Based on that, there is no "retry counter" nor "timeout amplify".
My understanding was based on the usage of this term in DTLSFlight.

If "retransmission" is a potential sequence of retransmissions, your right, then the terms are proper. 

May be "pauseRetransmissions" and "resumeRetransmissions" would also be possible terms?